servers/src/main/java/tachyon/master/block/BlockMaster.java

```diff
@@ -324,20 +334,18 @@ public void removeBlocks(List<Long> blockIds) {
   }
 
   /**
-   * @return a new block container id. Called by internal masters
+   * @return a new block container id
    */
   @Override
   public long getNewContainerId() {
-    synchronized (mBlockContainerIdGenerator) {
```

Comment:
this synchronization is not needed because BlockContainerIdGenerator is thread-safe